SCClassLibrary/Common/Collections/SequenceableCollection.sc

```diff
@@ -46,6 +46,16 @@ SequenceableCollection : Collection {
 		});
 		^obj
 	}
+	*recursiveFormula { arg a, size;
+		var i=0, divisor = 3, divisorDec = 0.9;
+		var obj = this.new(size);
```

Comment:
This will only work for subclasses of `SequenceableCollection` (e.g. `Array`). To make this clearer you could add a line `this.subclassResponsibility(thisMethod);` after the variable declaration.